pkg/networkservice/common/authorize/server.go

```diff
@@ -40,6 +40,18 @@ func NewServer(policies ...opa.AuthorizationPolicy) networkservice.NetworkServic
 	}
 }
 
+// NewServerWithDefaultPolicies -  returns a new authorization networkservicemesh.NetworkServiceServers with default OPA policies
+func NewServerWithDefaultPolicies(additionalPolicies ...opa.AuthorizationPolicy) networkservice.NetworkServiceServer {
```

Comment:
@edwarnicke Can we use this constructor for cmd-nsmgr, cmd-forwarder, cmd-icmp-responder?

Reply:
Rather than this (which breaks pattern) how about we introduce a With pattern here, and have the normal server default to these policies?

Reply:
good idea, fixed